lib/zomato.rb

```diff
@@ -0,0 +1,98 @@
+require 'net/http'
+
+$key = ENV['ZOMATO_API_KEY']
+$base_url = 'https://developers.zomato.com/api/v2.1'
+
+class Zomato
+
+  def initialize(city_id)
+    @city_id = city_id
+  end
+
+  def createCuisines
+    charset = Array(1..9) + Array('a'..'z')
+    uri = URI("#{$base_url}/cuisines?city_id=#{@city_id}")
+
+    Net::HTTP.start(uri.host, uri.port, :use_ssl => uri.scheme == 'https') do |http|
+      request = Net::HTTP::Get.new uri
+      request.add_field("user-key", $key)
+      request.add_field("accept", "application/json")
+      response = http.request request # Net::HTTPResponse object
+      if !response.is_a?(Net::HTTPSuccess)
```

Comment:
minor: `if !something` can be transformed into `unless something`

in this case if we didnt write an output then i would have done this in a guard clause:
`return unless something`